app/uk/gov/hmrc/thirdpartyapplication/models/db/ApplicationData.scala

```diff
@@ -41,7 +41,8 @@ case class ApplicationData(id: UUID,
                            rateLimitTier: Option[RateLimitTier] = Some(BRONZE),
                            environment: String = Environment.PRODUCTION.toString,
                            checkInformation: Option[CheckInformation] = None,
-                           blocked: Boolean = false) {
+                           blocked: Boolean = false,
+                           cidrBlocks: Set[String] = Set.empty) {
```

Comment:
Do we need to make any considerations for IPv6, as opposed to assuming IPv4?

Reply:
I don't think we support IPv6 on the platform

Reply:
Yeah, that was a bit of a gratuitous one. Just wondered whether we want to build in a way of representing them now, rather than having to retrofit it later. (Which we'll almost certainly never do, I suppose!)